File: sdkwork-backend-sdk-go/types/app_sdk_config_vo.go
package types

// App SDK配置
type AppSdkConfigVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Platform string `json:"platform"`
	OfficialAccount OfficialAccountSdkConfigVO `json:"officialAccount"`
	MiniProgram MiniProgramSdkConfigVO `json:"miniProgram"`
	App MobileAppSdkConfigVO `json:"app"`
}

File: sdkwork-backend-sdk-csharp/Models/AppSdkConfigVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class AppSdkConfigVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public string? Platform { get; set; }
        public OfficialAccountSdkConfigVO? OfficialAccount { get; set; }
        public MiniProgramSdkConfigVO? MiniProgram { get; set; }
        public MobileAppSdkConfigVO? App { get; set; }
    }
}
